src/cli.test.js

```diff
@@ -133,6 +177,32 @@ test("installs packages correctly even if package name ends with '-0'", t => {
   });
 });
 
+test("peer dependency is not installed when a later version already exists", t => {
+  resetFixtureNodeModules("has-newer-peer-dep")
+    .then(() => {
+      const cli = spawnCli("data-forge-indicators", "has-newer-peer-dep");
+      cli.on("exit", code => {
+        t.equal(code, 0, `errored, exit code was ${code}`);
```

Comment:
This might help with the tests in general -- if you add a third argument like

```js
t.equal(/\byarn global\b/.test(cmd), true, `errored, got ${cmd}`);
```

to the failing tests, you'll be able to see what's going on.

Reply:
I'm not sure what you mean.

Where does the `cmd` variable come from?

Reply:
It's from another test a few lines below:

```js
test("places `global` as first arg following `yarn` when using yarn and `--global` is specified", t => {
  getCliInstallCommand(["eslint-config-airbnb", "--global", "-Y"]).then(
    command => {
      const cmd = command.toString();
      t.comment(cmd);
      t.equal(/\byarn global\b/.test(cmd), true);
      t.end();
    },
    t.fail
  );
});
```